cartographer/mapping/map_builder_interface.h

```diff
@@ -78,7 +78,8 @@ class MapBuilderInterface {
                                     proto::SubmapQuery::Response* response) = 0;
 
   // Serializes the current state to a proto stream.
-  virtual void SerializeState(io::ProtoStreamWriterInterface* writer) = 0;
+  virtual void SerializeState(io::ProtoStreamWriterInterface* writer,
```

Comment:
What do you mean by unfinished? Submaps that have less than the configured number of range data insertions? Would be good to comment this here, because internal to PoseGraph is other state whether the submap is finished and ready for constraint search.